sklearn/preprocessing/tests/test_polynomial.py

```diff
@@ -552,6 +552,33 @@ def test_polynomial_features_csr_X(deg, include_bias, interaction_only, dtype):
     assert_array_almost_equal(Xt_csr.A, Xt_dense)
 
 
+@pytest.mark.parametrize("columns", [1, 2, 3, 1000])
+def test_polynomial_features_csr_wide(columns):
```

Comment:
I'd prefer docstrings to code comments for tests.

Reply:
done